README.md

```diff
@@ -117,32 +102,13 @@ Versioning is strictly based on [Semantic Versioning](https://semver.org/)
 * push commit and tag
 * upload rock to luarocks: `luarocks upload rockspecs/[name] --api-key=abc`
 
-### 2.0.0 (22-Sep-2020)
-
-* BREAKING: fallback for deprecated top-level field `type` is now removed
-  (deprecated since `0.5.0`) [#56](https://github.com/Kong/lua-resty-healthcheck/pull/56)
-* BREAKING: Bump `lua-resty-worker-events` dependency to `2.0.0`. This makes
-  a lot of the APIs in this library asynchronous as the worker events `post`
-  and `post_local` won't anymore call `poll` on a running worker automatically,
-  for more information, see:
-  https://github.com/Kong/lua-resty-worker-events#200-16-september-2020
-* BREAKING: tcp_failures can no longer be 0 on http(s) checks (unless http(s)_failures
-  are also set to 0) [#55](https://github.com/Kong/lua-resty-healthcheck/pull/55)
-* feature: Added support for https_sni [#49](https://github.com/Kong/lua-resty-healthcheck/pull/49)
-* fix: properly log line numbers by using tail calls [#29](https://github.com/Kong/lua-resty-healthcheck/pull/29)
-* fix: when not providing a hostname, use IP [#48](https://github.com/Kong/lua-resty-healthcheck/pull/48)
-* fix: makefile; make install
-* feature: added a status version field [#54](https://github.com/Kong/lua-resty-healthcheck/pull/54)
-* feature: add headers for probe request [#54](https://github.com/Kong/lua-resty-healthcheck/pull/54)
-* fix: exit early when reloading during a probe [#47](https://github.com/Kong/lua-resty-healthcheck/pull/47)
-* fix: prevent target-list from being nil, due to async behaviour [#44](https://github.com/Kong/lua-resty-healthcheck/pull/44)
-* fix: replace timer and node-wide locks with resty-timer, to prevent interval
-  skips [#59](https://github.com/Kong/lua-resty-healthcheck/pull/59)
-* change: added additional logging on posting events [#25](https://github.com/Kong/lua-resty-healthcheck/issues/25)
-* fix: do not run out of timers during init/init_worker when adding a vast
-  amount of targets [#57](https://github.com/Kong/lua-resty-healthcheck/pull/57)
-* fix: do not call on the module table, but use a method for locks. Also in
-  [#57](https://github.com/Kong/lua-resty-healthcheck/pull/57)
+### 3.0.0 (12-Oct-2023)
+
+* Perf: optimize by localizing some functions [#92](https://github.com/Kong/lua-resty-healthcheck/pull/92) (backport)
+* Fix: Generate fresh default http_statuses within new() [#83](https://github.com/Kong/lua-resty-healthcheck/pull/83) (backport)
+
+### 2.0.0
```

Comment:
It seems to me we can keep the module's history.

Reply:
I've kept the history but added a note to make sure nobody expects changes from 2.0.0 to automatically be in 3.0.0